src/admin-views/admin-welcome-message.php

```diff
@@ -2,84 +2,213 @@
 ?>
-</p>
 
+<?php if ( $has_plus ) : ?>
+	<div class="tribe-events-admin-notice">
+		<div class="tribe-events-admin-content-wrapper">
+			<img
+				class="tribe-events-admin-notice__logo"
+				src="<?php echo esc_url( tribe_resource_url( 'images/icons/horns-white.svg', false, null, $main ) ); ?>"
+				alt="<?php esc_attr_e( 'The Events Calendar product suite logo', 'event-tickets' ); ?>"
+			/>
+			<p><strong><?php echo esc_html_x( 'WHOHOO!', 'short expression of excitement', 'event-tickets' ); ?></strong> <?php esc_html_e( 'You\'re the proud owner of Event Tickets Plus! Let\'s get started…', 'event-tickets' ); ?></p>
+		</div>
+	</div>
+<?php endif; ?>
+
+<div class="tribe-events-admin-content-wrapper tribe-events-admin-tickets <?php if ( $has_plus ) { echo 'tribe-events-admin-tickets-plus'; } ?>">
+	<img
+		class="tribe-events-admin-graphic tribe-events-admin-graphic--desktop-only"
+		<?php $desktop_graphic = $has_plus ? 'images/header/welcome-desktop-etplus.jpg' : 'images/header/welcome-desktop-et.jpg'; ?>
+		src="<?php echo esc_url( tribe_resource_url( $desktop_graphic, false, null, $main ) ); ?>"
+		alt="<?php esc_attr_e( 'Shapes and lines for visual interest', 'event-tickets' ); ?>"
+	/>
 
-<div class="tribe-row">
+	<img
+		class="tribe-events-admin-graphic tribe-events-admin-graphic--mobile-only"
+		<?php $mobile_graphic = $has_plus ? 'images/header/welcome-mobile-etplus.jpg' : 'images/header/welcome-mobile-et.jpg'; ?>
+		src="<?php echo esc_url( tribe_resource_url( $mobile_graphic, false, null, $main ) ); ?>"
+		alt="<?php esc_attr_e( 'Shapes and lines for visual interest', 'event-tickets' ); ?>"
+	/>
 
-	<div class="tribe-half-column">
-		<h4 data-tribe-icon="dashicons-welcome-learn-more"><?php esc_html_e( 'Getting Started', 'event-tickets' ); ?></h4>
-		<p><?php esc_html_e( 'Start strong with these helpful resources.', 'event-tickets' ); ?></p>
-		<ul>
-			<li><a href="https://theeventscalendar.com/product/wordpress-event-tickets/" target="_blank"><?php esc_html_e( 'Key Features', 'event-tickets' ); ?></a></li>
-			<li><a href="http://m.tri.be/1a9u" target="_blank"><?php esc_html_e( 'Settings Overview', 'event-tickets' ); ?></a></li>
-			<li><a href="http://m.tri.be/1a9v" target="_blank"><?php esc_html_e( 'New User Primer', 'event-tickets' ); ?></a></li>
-		</ul>
+	<div class="tribe-events-admin-title">
+		<img
+			class="tribe-events-admin-title__logo"
+			<?php $logo_image = $has_plus ? 'images/logo/event-tickets-plus.svg' : 'images/logo/event-tickets.svg'; ?>
+			src="<?php echo esc_url( tribe_resource_url( $logo_image, false, null, $main ) ); ?>"
+			alt="<?php esc_attr_e( 'Event Tickets logo', 'event-tickets' ); ?>"
+		/>
+		<h2 class="tribe-events-admin-title__heading"><?php
+			if ( $has_plus ) :
+				esc_html_e( 'Event Tickets Plus', 'event-tickets' );
+			else :
+				esc_html_e( 'Event Tickets', 'event-tickets' );
+			endif;
+		?></h2>
+		<p class="tribe-events-admin-title__description"><?php
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 1a57267929a650080835f1b3c25dd7ce</sub>